### PR TITLE
feat: use vercel (#1542)

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [x] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [ ] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
**Vercel COMMING!!!**

### CODE REVIEW

**Review by WANZARGEN — APPROVED** [review]
LGTM!

**Review by sulmoJ — APPROVED** [review]
Yeaaaaaaah

File: package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "lint": "turbo run lint --parallel",
     "format": "turbo run format",
     "test": "NODE_OPTIONS=--max_old_space_size=8192 turbo run test --parallel",
-    "prepare": "node -e \"if(require('fs').existsSync('.git')){process.exit(1)}\" || is-ci || husky install"
+    "prepare": "node -e \"if(require('fs').existsSync('.git')){process.exit(1)}\" || npx husky install"
```
Comment (piggggggggy) [inline package.json:17]: 😧

**Review by piggggggggy — APPROVED** [review]
LGTM!

**Pull request merged**